C# cdktf get fails on windows, symlink

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### cdktf & Language Versions

0.1.0 (2 days old)

### Affected Resource(s)

<!--- Please list the affected resources. --->

### Debug Output

https://gist.github.com/mikeparker/0a4aaa37c1d31efdf2956f40b5e1045d
In particular `Error: EPERM: operation not permitted, symlink ...`

### Expected Behavior

It completes

### Actual Behavior

It errored

### Steps to Reproduce

Run ` cdktf get` on windows

@jsteinich yes, running as admin solves this. As a quick intermediate workaround it might be worth just adding a note to the readme that admin privs are needed on windows machines (not all developers have admin access).

After that it runs into whitespace directory name errors on my machine, same as with some other commands
```
C:\Git\src\github.com\mikeparker\hello-terraform2> cdktf get
⠇ downloading and generating providers...
⠏ downloading and generating providers...
Error: jsii compilation failed with non-zero exit code: 1
  | Error: Command (npm pack C:\Users\Mike Parker\AppData\Local\Temp\temp-KCq1q5) failed with status 1:
  | #STDERR> npm ERR! code ENOLOCAL
  | #STDERR> npm ERR! Could not install from "..\..\..\..\..\Mike" as it does not contain a package.json file.
```
Shall I open a separate ticket for that?

--- a/packages/cdktf-cli/package.json
+++ b/packages/cdktf-cli/package.json
@@ -38,7 +38,7 @@
     "ink": "^2.7.1",
     "ink-confirm-input": "^2.0.0",
     "ink-spinner": "^3.0.1",
-    "jsii-srcmak": "^0.1.195",
+    "jsii-srcmak": "^0.1.223",
     "log4js": "^6.3.0",
     "open": "^7.0.4",
     "react": "^16.13.1",

